tools/rapids-get-pr-conda-artifact-github

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Echo path to an artifact for a specific PR from GitHub Artifacts.
+# Finds and uses the latest commit on the PR.
+#
+# Positional Arguments:
+#   1) repo name
+#   2) PR number
+#   3) "cpp" or "python", to get the artifact for the C++ or Python build, respectively
+#   4) [optional] commit hash, to get the artifact for a specific commit
+#
+# Example Usage:
+#   rapids-get-pr-conda-artifact-github rmm 1095 cpp
+set -euo pipefail
+
+repo="${1}"
+pr="${2}"
+package_type="${3}"
+commit="${4:-}"
+
+# Create a temporary directory for the artifact
+unzip_dest=$(mktemp -d)
```

Comment:
Could we add this as an optional additional argument for dev use to pull the packages down elsewhere?  Or alternatively, because argument munging in bash is hard, we could support setting an env-var to determine download location?

Reply:
@gforsyth, so something like this?

```sh
unzip_dest="${RAPIDS_UNZIP_DIR:-$(mktemp -d)}"
```

Reply:
yeah, exactly